models.py

```diff
@@ -40,13 +40,15 @@ class Recipe(db.Model):
     Recipe Model
     """
     __tablename__ = 'recipe'
+    __searchable__ = ['title']
+
     id = Column(Integer, autoincrement=True, primary_key=True)
     title = Column(String(250), default='')
     time_needed = Column(Integer, default=15)
     ingredients = Column(String(5000), default='')
     # ingredients = Column(JSON, default='')
     steps = Column(String(5000), default='')
-    is_public = Column(Boolean, default=True)
+    status = Column(String, default='Public', nullable=False)
```

Comment:
Wydaje mi się, że ta zmiana jest niepotrzebna - przepis będzie tylko w stanie prywatnym/publicznym i atrybut is_public typu bool nam to załatwia.Widziałam, że to może mieć związek z tym, że formularz nie obsługuje typu bool, ale w takim razie konwersja string -> bool mogłaby się odbywać w trakcie parsowania formularza.

Reply:
tak to działało, ale to było strasznie nahakowane, bo string konwertowany do booleana zawsze zwraca True. also, z booleanem tak konwertowanym nie działa metoda form.validate()